mlflow/server/handlers.py

```diff
@@ -468,6 +469,32 @@ def _get_model_registry_store(registry_store_uri: str | None = None) -> Abstract
     return _model_registry_store
 
 
+def _get_job_store(backend_store_uri: str | None = None) -> "SqlAlchemyJobStore":
```

Comment:
```suggestion
def _get_job_store(backend_store_uri: str | None = None) -> "AbstractJobStore":
```

We should not return interface, not implementation